### PR TITLE
Reformat some tooltips

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: EnglishTranslation-master/SKILL.tsv [hunk 1]
```diff
@@ -4129,7 +4129,7 @@ SKILL_20150714_004128	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRati
 SKILL_20150714_004129	Removes the armor's effect from the target.
 SKILL_20150714_004130	Decreases the target's defense.
 SKILL_20150714_004131	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}AoE Attack Ratio: #{SkillSR}#
-SKILL_20150714_004132	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Ignores the defense of small-type and medium-type targets #{CaptionRatio}#{nl}AoE Attack Ratio: #{SkillSR}#
+SKILL_20150714_004132	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Defense Ignored: #{CaptionRatio}#{nl}AoE Attack Ratio: #{SkillSR}#
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv:4132]: You removed the part where it says it ignores the defense of **small and medium-sized/type targets**. This should remain in the tooltip for clarification.

File: EnglishTranslation-master/SKILL.tsv [hunk 2]
```diff
@@ -5379,8 +5379,8 @@ SKILL_20151022_005378	Mijin no Jutsu
 SKILL_20151022_005379	Bunshin no Jutsu
 SKILL_20151022_005380	Mokuton no Jutsu
 SKILL_20151022_005381	Katon no Jutsu
-SKILL_20151022_005382	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Cast an arrow of ice that inflicts damage and has a chance of freezing an enemy.
-SKILL_20151022_005383	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Make ice spikes sprout out in a line, inflicting damage and freezing enemies by chance.
+SKILL_20151022_005382	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Cast an arrow of ice that inflicts damage and has a chance of freezing the enemy.
+SKILL_20151022_005383	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Make ice spikes sprout out in a line and inflict damage. Has a chance of freezing enemies.
```
Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: ...a line and inflict damage -> ...a line to inflict damage

File: EnglishTranslation-master/SKILL.tsv (hunk 1 above)
Comment (desuwa) [inline EnglishTranslation-master/SKILL.tsv]: It is still in the description: https://github.com/Treeofsavior/EnglishTranslation/blob/master/EnglishTranslation-master/SKILL.tsv#L3612.

The line I modified is the section where actual numbers are displayed.

Comment (ghost) [inline EnglishTranslation-master/SKILL.tsv:4132]: It's in the description: https://github.com/Treeofsavior/EnglishTranslation/blob/master/EnglishTranslation-master/SKILL.tsv#L3612

The modified line is part of the section where actual values are displayed.

File: EnglishTranslation-master/SKILL.tsv [hunk 3]
```diff
@@ -4933,7 +4933,7 @@ SKILL_20150729_004932	Uses the highest abilities between you and allies linked b
 SKILL_20150729_004933	Shrink the size of your target.
 SKILL_20150729_004934	Enlarge the size of your target. Doubles the target's maximum HP, EXP and drop rate.
 SKILL_20150729_004935	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Duration: 10 seconds
-SKILL_20150729_004936	Duration: #{CaptionTime}# seconds{nl}Chance of doubling the duration: #{CaptionRatio}#%{nl}Movement Speed: -10%{nl}{#DD5500}{ol}Lightning{/}{/}Property Additional Damage: +35%{nl}{#DD5500}{ol}Flame{/}{/}Property Target Attack: #{SkillAtkAdd}#
+SKILL_20150729_004936	Duration: #{CaptionTime}# seconds{nl}Chance of doubling the duration: #{CaptionRatio}#%{nl}Movement Speed: -10%{nl}{#DD5500}{ol}Lightning{/}{/} Property Additional Damage: +35%{nl}{#DD5500}{ol}Flame{/}{/} Property Attack: #{SkillAtkAdd}#
```
Comment (grillo-delmal) [inline EnglishTranslation-master/SKILL.tsv]: You added an extra space in between "`Flame{/}{/}`" and "`Property`", the `{/}{/}` already adds a space between the two words. Was that intentional?

Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: {#DD5500}{ol}Lightning{/}{/} Property Additional Damage: +35% 
to
Additional {#DD5500}{ol}Lightning{/}{/} Property Damage: +35%

The original does mention 'target' in the last. Does this skill increase your Flame property attack?  Is it a debuff? I don't know how this skill works.

Comment (ghost) [inline EnglishTranslation-master/SKILL.tsv]: > the {/}{/} already adds a space between the two words

I wasn't sure about this. I saw plenty of lines with spaces after {/}{/}

Comment (ghost) [inline EnglishTranslation-master/SKILL.tsv]: It should work a Fire DOT as far as I know. For a 5/5 Rain, the tooltip gives a value of something like 550. I guess _Flame_ should be _Fire_ in that case.
Somebody should confirm to make sure.

File: EnglishTranslation-master/SKILL.tsv [hunk 4]
```diff
@@ -4933,7 +4933,7 @@ SKILL_20150729_004932	Uses the highest abilities between you and allies linked b
 SKILL_20150729_004933	Shrink the size of your target.
 SKILL_20150729_004934	Enlarge the size of your target. Doubles the target's maximum HP, EXP and drop rate.
 SKILL_20150729_004935	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Duration: 10 seconds
-SKILL_20150729_004936	Duration: #{CaptionTime}# seconds{nl}Chance of doubling the duration: #{CaptionRatio}#%{nl}Movement Speed: -10%{nl}{#DD5500}{ol}Lightning{/}{/}Property Additional Damage: +35%{nl}{#DD5500}{ol}Flame{/}{/}Property Target Attack: #{SkillAtkAdd}#
+SKILL_20150729_004936	Duration: #{CaptionTime}# seconds{nl}Chance of doubling the duration: #{CaptionRatio}#%{nl}Movement Speed: -10%{nl}Additional {#DD5500}{ol}Lightning{/}{/}Property Damage: +35%{nl}{#DD5500}{ol}Fire{/}{/}Property Attack: #{SkillAtkAdd}#
```
Comment (ghost) [inline EnglishTranslation-master/SKILL.tsv]: Worked like a DOT in icbt2. No idea how it is now, though.
hxxps://www.youtube.com/watch?v=xFXACL00a9E

Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: @zhouyu47 just tried it out. It only affects fire property mobs. 
It should be changed to: **Attack on Fire Property Targets** - That would be better.

Comment (ghost) [inline EnglishTranslation-master/SKILL.tsv]: Thank you.
Shouldn't it be something other than "property" for monsters? Element, or attribute maybe?

Comment (Sunaries) [inline EnglishTranslation-master/SKILL.tsv]: Attribute is already a term for the skills. IMC coined "element" as property in this game.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Pull request merged**